docs/antrea-ipam.md

```diff
@@ -470,13 +468,40 @@ spec:
     prefixLength: 64
 ```
 
-VLAN ID in the IP range subnet definition of `IPPool` CRD is not supported for
-secondary network IPAM.
+When used for Antrea VLAN secondary network, the VLAN set in an `IPPool` IP
+range will be passed to the VLAN secondary interface configuration. For example:
 
-### Secondary Network creation with Multus
+```yaml
+apiVersion: "crd.antrea.io/v1alpha2"
+kind: IPPool
+metadata:
+  name: ipv4-pool-1
+spec:
+  ipVersion: 4
+  ipRanges:
+  - cidr: "10.10.1.0/26"
+    gateway: "10.10.1.1"
+    prefixLength: 24
+    vlan: 100
+
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ipv4-net-1
+spec:
+  {
+      "cniVersion": "0.3.0",
+      "type": "antrea",
+      "networkType": "vlan",
+      "ipam": {
+          "type": "antrea",
+          "ippools": [ "ipv4-pool-1" ]
+      }
+  }
+```
+
+You can refer to the [Antrea secondary network document](secondary-network.md)
+for more information about Antrea secondary VLAN network configuration.
 
-To leverage Antrea for secondary network IPAM, Antrea must be used as the CNI
-for the Pods' primary network, while the secondary networks are implemented by
-other CNIs which are managed by Multus. The [Antrea + Multus guide](cookbooks/multus)
-talks about how to use Antrea with Multus, including the option of using Antrea
-IPAM for secondary networks.
+For other network types, the VLAN field will be ignored.
```

Comment:
> Antrea IPAM code returns the first non-zero VLAN in the configured IPPool subnets.

But then shouldn't we have IPPool validation to prevent users from using different VLAN values in the same IPPool, or is there a use case for that?